components/schema-migrator/migrations/director/202012081537_ord-service-initial.up.sql

```diff
@@ -0,0 +1,302 @@
+      FROM api_definitions,
+           jsonb_to_recordset(api_definitions.api_definitions) AS api_res_defs(type TEXT, "customType" TEXT,
+                                                                               "mediaType" TEXT,
+                                                                               url TEXT)) as api_defs
+UNION ALL
+(SELECT id                                  AS api_definition_id,
+        CASE
+            WHEN spec_type::text = 'ODATA' THEN 'edmx'
+            WHEN spec_type::text = 'OPEN_API' THEN 'openapi-v3'
+            ELSE spec_type::text
+            END                             AS type,
+        NULL                                AS custom_type,
+        format('/api/%s/specification', id) AS url,
+        CASE
+            WHEN spec_format::text = 'YAML' THEN 'text/yaml'
+            WHEN spec_format::text = 'XML' THEN 'application/xml'
+            WHEN spec_format::text = 'JSON' THEN 'application/json'
+            ELSE spec_format::text
+            END                             AS media_type
+ FROM api_definitions);
+
+CREATE VIEW api_resource_links AS
+SELECT id                   AS api_definition_id,
+       actions.type         AS type,
+       actions."customType" AS custom_type,
+       actions.url          AS url
+FROM api_definitions,
+     jsonb_to_recordset(api_definitions.api_resource_links) AS actions(type TEXT, "customType" TEXT, url TEXT);
+
+CREATE VIEW changelog_entries AS
+SELECT *
+FROM (SELECT id                      AS api_definition_id,
+             NULL::uuid              AS event_definition_id,
+             entries.version         AS version,
+             entries."releaseStatus" AS release_status,
+             entries.date            AS date,
+             entries.description     AS description,
+             entries.url             AS url
+      FROM api_definitions,
+           jsonb_to_recordset(api_definitions.changelog_entries) AS entries(version TEXT, "releaseStatus" TEXT,
+                                                                            date TEXT,
+                                                                            description TEXT, url TEXT)) AS api_entries
+UNION ALL
+(SELECT NULL::uuid              AS api_definition_id,
+        id                      AS event_definition_id,
+        entries.version         AS version,
+        entries."releaseStatus" AS release_status,
+        entries.date            AS date,
+        entries.description     AS description,
+        entries.url             AS url
+ FROM event_api_definitions,
+      jsonb_to_recordset(event_api_definitions.changelog_entries) AS entries(version TEXT, "releaseStatus" TEXT,
+                                                                             date TEXT,
+                                                                             description TEXT, url TEXT));
+
+CREATE VIEW event_resource_definitions AS
```

Comment:
Currently each Event has only a single Event Definition, once we implement the aggregator this view should be fully represented in a separate specifications table which will have the data as well